simapp/test_helpers.go

```diff
@@ -23,32 +23,37 @@ import (
 // Setup initializes a new SimApp. A Nop logger is set in SimApp.
 func Setup(isCheckTx bool) *SimApp {
 	db := dbm.NewMemDB()
-	app := NewSimApp(log.NewNopLogger(), db, nil, true, 0)
+	app := NewSimApp(log.NewNopLogger(), db, nil, true, []int64{}, 0)
```

Comment:
I was thinking the extra arg to `NewSimApp` would be a breaking change to many modules, but sure, other modules use `Setup` in test code. Seems fine.